src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
-}
+  static alive = []
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
 
+    if (this.health) {
```

Comment:
I guess this check can be redundant, because `this.health` always contains some value